src/grammar.json

```diff
@@ -738,8 +738,124 @@
     ],
 
     "card": [
```

Comment:
also, I think we'll want to update the #card# usages—I think some of the headlines work better with #cardalone# rather than including #descriptiveCard#, since it tends to refer to a specific card, rather than a canonical one.